src/main/java/com/integerlimit/deprecatedapi/api/DeprecatedBlocks.java

```diff
@@ -2,41 +2,37 @@
 
 
 import com.integerlimit.deprecatedapi.DeprecatedAPI;
-import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
 import net.minecraft.block.Block;
-import net.minecraft.item.Item;
 import net.minecraft.util.ResourceLocation;
-import org.apache.commons.lang3.tuple.Pair;
 
 import javax.annotation.Nullable;
-import java.util.Map;
+import java.util.ArrayList;
+import java.util.List;
 
 public class DeprecatedBlocks {
-    private static final Map<Pair<Block, Integer>, DeprecatedBlock> blocks = new Object2ObjectOpenHashMap<>();
-    public static final int WILDCARD_META = -1;
```

Comment:
I would prefer storing the wildcard here too, so if others are using it, then they can get it from this class, instead of having to get it from DeprecatedItems.